addons/mail/static/src/core/common/message_actions.js

```diff
@@ -183,11 +183,11 @@ registerMessageAction("download_files", {
     sequence: 55,
 });
 registerMessageAction("toggle-translation", {
-    condition: ({ message, thread }) => message.isTranslatable(thread),
-    icon: ({ owner }) =>
-        `fa fa-language ${owner.state.showTranslation ? "o-mail-Message-translated" : ""}`,
-    name: ({ owner }) => (owner.state.showTranslation ? _t("Revert") : _t("Translate")),
-    onSelected: ({ owner }) => owner.onClickToggleTranslation(),
+    condition: ({ message }) => message.isTranslatable(message.thread),
```

Comment:
Note that `isTranslatable` could have no param, since this is on `message` and the param is always deduced from `message` (`message.thread`)